src/components/App.js

```diff
@@ -1,21 +1,32 @@
-import React, { PureComponent } from 'react';
+import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+
 import calculate from '../logic/calculate';
 
-class App extends PureComponent {
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: '0',
+      next: '0',
+      operation: '',
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick= btnName => {
+    const result = calculate(this.state, btnName);
+    this.setState(result);
+  }
+
   render() {
+    const { total, operation, next } = this.state;
     return (
       <>
-        <Display result={calculate(
-          {
-            total: 10,
-            next: 12,
-            operation: '',
-          }, '+',
-        ).operation}
-        />
-        <ButtonPanel />
+        <Display result={`${total}${operation}`} />
+        <Display result={`${next}`} />
```

Comment:
`App` should render a single `Display` and pass the `total` __or__ `next` to it from the state. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/passToatalToDisplay.png))